locale/fr/get-involved/index.md

```diff
@@ -3,44 +3,48 @@ title: Comment s'impliquer ?
 
-- Le compte Twitter officiel de Node.js est [nodejs](https://twitter.com/nodejs) [en].
+- Le compte Twitter officiel de Node.js est [nodejs](https://twitter.com/nodejs) (en anglais).
 
-- Le [calendaire de la Fondation Node.js](https://nodejs.org/calendar) avec toutes les réunions d'équipe publiques.
+- Le [calendrier de la Fondation Node.js](https://nodejs.org/calendar) avec toutes les réunions publiques d'équipe (en anglais).
 
-- [Node.js Everywhere](https://newsletter.nodejs.org) est la newsletter mensuelle officielle de Node.js [en].
+- [Node.js Everywhere](https://newsletter.nodejs.org) est la _newsletter_ mensuelle officielle de Node.js (en anglais).
 
-- [Node.js Collection](https://medium.com/the-node-js-collection) est une liste de contenus maintenue par la communauté sur Medium [en].
+- [Node.js Collection](https://medium.com/the-node-js-collection) est une liste de contenus maintenue par la communauté sur Medium (en anglais).
 
-- [NodeUp](http://nodeup.com) est un podcast sur les actualités de la communauté Node [en].
+- [NodeUp](http://nodeup.com) est un podcast sur les actualités de la communauté Node (en anglais).
 
-- La [Community Committee](https://github.com/nodejs/community-committee) s'agit d'un comité de haut niveau de la Fondation Node.js axé sur les efforts communautaires.
+- La [Community Committee](https://github.com/nodejs/community-committee) s'agit d'un comité de haut niveau de la Fondation Node.js axé sur les efforts communautaires (en anglais).
 
 
 ## Apprentissage
 
-- [Documentation officiel de l'API de référence](/api) détaille l'API Node [en].
+- [La documentation officielle de l'API de référence](/api) détaille l'API Node (en anglais).
 
-- [NodeSchool.io](http://nodeschool.io) vous apprendra les concepts de Node.js avec des jeux intéractifs en ligne de commande [en].
+- [NodeSchool.io](https://nodeschool.io/fr-fr/) vous apprendra les concepts de Node.js avec des jeux interactifs en ligne de commande.
 
-- [L'étiquette Stack Overflow Node.js](http://stackoverflow.com/questions/tagged/node.js) rassemble de nouvelles informations chaque jours [en].
+- [L'étiquette Stack Overflow Node.js](http://stackoverflow.com/questions/tagged/node.js) rassemble de nouvelles informations chaque jours (en anglais).
 
-- [How To Node](http://howtonode.org/) a de nombreux tutoriaux utiles [en].
+- [L'étiquette Node.js de la communauté DEV](https://dev.to/t/node) est un endroit pour partager ses projets Node.js, des articles et des tutoriaux mais aussi pour démarrer des conversations et demander des retours sur des sujets liés à Node.js. Tous les niveaux de connaissances sont acceptés. (en anglais)
+
+- [Nodeiflux](https://discordapp.com/invite/vUsrbjd) est une communauté sympathique d'entraide entre développeu·r·se·s _back-end_ Node.js sur le service Discord (en anglais).
```

Comment:
ditto

Reply:
I would prefer we find a way of saying that without needing this kind of writing.
Like " ...sympathique centrée sur le developpement _back-end_ avec Node.js sur ..."